src/Components/Alert/Color_Options_Manager.php

```diff
@@ -0,0 +1,18 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Alert\Components\Alert;
+
+interface Color_Options_Manager {
+
+	/**
+	 * Should return multidimensional array where key is HEX color and value is array of a color name and class name.
+	 *
+	 * @return array
+	 */
+	public function get_all_options(): array;
+
+	public function get_acf_options(): array;
```

Comment:
Please add docBlocks to these so developers know how to properly implement this interface.

Reply:
Added